### ISSUE TITLE
New update broke the module

### ISSUE THREAD

**Issue by leagueoffoundryvttdevs** [League-of-Foundry-Developers/fvtt-module-lmrtfy#66]
**Submitted By**: nulmas@hotmail.com
**Feedback Type**: Bug
 
**Core:** 0.7.9
**System:** dnd5e v1.2.4
**Module Version:** lmrtfy v1.9
 
**Description**:
The new update made the module stop working (at least in 5e, on native application).

Even if it's the only moduled active, the screen won't appear.

**Comment by akrigline** [comment]
some more details from discord reports:
```
foundry.js:4481 TypeError: An error occurred while rendering LMRTFYRequestor 88: LMRTFY.skills.sort is not a function
    at LMRTFYRequestor.getData (requestor.js:30)
    at LMRTFYRequestor._render (foundry.js:4509)
    at LMRTFYRequestor._render (foundry.js:5157)
    at LMRTFYRequestor.render (foundry.js:4479)
    at LMRTFYRequestor.render (requestor.js:57)
    at Function.requestRoll (lmrtfy.js:109)
    at Object.onClick (lmrtfy.js:133)
    at SceneControls._onClickTool (foundry.js:19950)
    at HTMLLIElement.dispatch (jquery.min.js:2)
    at HTMLLIElement.v.handle (jquery.min.js:2)
```

**Comment by akrigline** [comment]
Fixed in 1.10
